Cargo.toml

```diff
@@ -26,10 +26,10 @@ toml = "0.5.8"
 serde = { version = "1.0.123", features = ["derive"] }
 env_logger = "0.8.3"
 log = { version = "0.4.14", features = ["serde"] }
-cryptoki = { version = "0.1.0", optional = true, features = ["psa-crypto-conversions"] }
+cryptoki = { git = "https://github.com/parallaxsecond/rust-cryptoki", rev = "850b826b631df354553bf62757f35cd394b3dfff", optional = true, features = ["psa-crypto-conversions"] }
```

Comment:
Why not crate 0.1.1?

Reply:
`0.1.1` does not contain https://github.com/parallaxsecond/rust-cryptoki/pull/18 which has some changes needed to fix #379